src/views/notification/View.vue

```diff
@@ -0,0 +1,133 @@
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Title
+                              </label>
+                              <input
+                                id="update-title-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.title != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Title"
+                                v-model="this.notification.title"
+                                disabled
+                              />
+                              <div v-if="this.validation_error?.title != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.title[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Content
+                              </label>
+                              <textarea
+                                id="update-content-notification"
+                                :class="'form-control' + (this.validation_error?.content != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Content"
+                                rows="10"
+                                v-model="this.notification.content"
+                                disabled
+                              />
+                              <div v-if="this.validation_error?.content != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.content[0] }}
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- END: Notification Latest Tab -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+import axios from 'axios'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+export default defineComponent({
+  data() {
+    return {
+      active_tab: false,
+      validation_error: {},
+      notification: {}
+    }
+  },
+  mounted() {
+    this.fetchNotification()
+  },
+  methods: {
+    fetchNotification() {
+      axios.get('notifications/' + this.$route.params.id)
+        .then(response => {
+          this.notification = response.data.data
+        })
+        .catch(error => {
+          console.error(error.response)
```

Comment:
Remove debug messages if not required constantly

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/315b3893315da80d9f5ff8fc43e6280f8bacd5ef